rotary/rules/rotary.smk

```diff
@@ -221,17 +217,45 @@ rule build_gtdb_mash_ref_database:
         """
 
 
+rule set_up_sample_directories:
+    input:
+        "samples.tsv"
+    output:
+        long_reads = expand("{sample}/{sample}_long.fastq.gz", sample=SAMPLE_NAMES),
```

Comment:
I think it might help for file organization to move these raw input files to a subfolder within {samples}, e.g., `{samples}/raw`. We might also consider setting the `temp()` flag on these. What do you think?

Reply:
@jmtsuji Not that, in most cases, these files will be symlinked, so they won't take up any disk space. However, yes, I think moving them to a `raw` folder makes sense. I like having a pointer to the raw files for QC/Debugging purposes.

Reply:
@jmtsuji This will effect your pathing for some of the rules where you `cd` back and forth.

Reply:
@LeeBergstrand OK, let's use a `raw` folder then. I'll see during end-to-end testing if anything goes awry with rules that need to `cd` back and forth

Reply:
Looks OK so far based on a quick code review of the multi pathing branch.